src/test/java/io/kestra/plugin/ai/rag/ChatCompletionTest.java

```diff
@@ -163,4 +165,40 @@ void rag_givenTavilyWebSearch() throws Exception {
         var ragOutput = rag.run(runContext);
         assertThat(ragOutput.getCompletion()).isNotNull();
     }
+
+    @EnabledIfEnvironmentVariable(named = "RAPID_API_KEY", matches = ".*")
+    @Test
+    void tools_givenJudge0JavaScript() throws Exception {
+        RunContext runContext = runContextFactory.of(Map.of(
+            "modelName", "tinydolphin",
+            "endpoint", ollamaEndpoint,
+            "apikey", "1e3b194adamshae3a588f676c08ep1d9f6ejsna0ce4f7eb0ea", // RAPID_API_KEY,
+            "flow", Map.of("id", "flow", "namespace", "namespace")
+        ));
+
+        var chat = ChatCompletion.builder()
+            .chatProvider(
+                Ollama.builder()
+                    .type(Ollama.class.getName())
+                    .modelName(Property.ofExpression("{{ modelName }}"))
+                    .endpoint(Property.ofExpression("{{ endpoint }}"))
+                    .build()
+            )
+            .embeddings(KestraKVStore.builder().build())
+            .prompt(Property.ofValue(
+                "Use the JavaScript execution tool to evaluate `21*2`. Return only the number."
```

Comment:
If it's a JavaScriptExecution tool let's call it `JavaScriptCodeExecution` (or skip the Code in it).

Reply:
No it's a global code executor tool but here it's just for the sake of the example with JavaScript. It allows also a bunch of languages.